.github/actions/deploy-happy-stack/action.yml

```diff
@@ -57,7 +57,7 @@ runs:
         HAPPY_DEPLOYMENT_FLAGS=""
 
         if [[ ${OPERATION} == "create-or-update" ]]; then
-          if $(happy list --aws-profile "" --env=${ENV} 2>&1 | grep -q "^|\s*${STACK_NAME}\s*|" ); then
+          if $(happy list --aws-profile "" --env=${ENV} &>out.txt; cat out.txt | grep "^|\s*${STACK_NAME}\s*|" ); then
```

Comment:
I think `grep 'search_string' out.txt` is canonical here.

Reply:
also -- wow. most `happy` commands should probably have a json output option so we can find the data we're looking for with `jq` -- this is error prone (we could match the table header row for example)